includes/discovery/sensors/state/panos.inc.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+$temp = snmp_get($device, "panSysHAMode.0", "-Ovqe", "PAN-COMMON-MIB");
+$cur_oid = '.1.3.6.1.4.1.25461.2.1.2.1.13.0';
+$index = '0';
+
+#Convert string to integer
+if ($temp == 'active-passive') {
+    $temp = 1;
+} elseif ($temp == 'active-active') {
+    $temp = 2;
+}
+
+if (is_numeric($temp)) {
+    //Create State Index
+    $state_name = 'panSysHAMode';
+    $state_index_id = create_state_index($state_name);
+
+    //Create State Translation
+    if ($state_index_id !== null) {
+        $states = array(
+            array($state_index_id,'Active-Passive',0,1,0) ,
+            array($state_index_id,'Active-Active',0,2,0)
+        );
+        foreach ($states as $value) {
```

Comment:
@FTBZ This isn't the most recent way to manage states in php.  If you can't get it to work in yaml, can you update to match the example here: https://docs.librenms.org/#Developing/Sensor-State-Support/